lib/formatters.js

```diff
@@ -134,14 +134,32 @@ function colorize(color, str) {
  * @deprecated
  */
 function formatTrace(level, context, message, args, err) {
-  args.unshift(
-      templateTrace + message,
-      (new Date()).toISOString(),
-      level,
-      context.corr || notAvailable,
-      context.trans || notAvailable,
-      context.op || notAvailable
-  );
+
+  var recontext = {
+    time: (new Date()).toISOString(),
+    lvl: level,
+    corr: context.corr || notAvailable,
+    trans: context.trans || notAvailable,
+    op: context.op || notAvailable
+  };
+
+  Object.keys(context)
+      .filter(function(key) {
+        return !(context[key] && Object.prototype.toString.call(context[key]) === '[object Function]');
```

Comment:
BTW, since the `filter()` code is executed on the results of `Object.keys()`, the `context[key]` check could also be avoided, this is: `return typeof context[key] === 'function';` should be more than enough... ;)